Centaurus/Alpha/AlphaStartup.cs

```diff
@@ -0,0 +1,165 @@
+using Microsoft.Extensions.Hosting;
+using NLog;
+using System.Linq;
+using System.Threading.Tasks;
+using Centaurus.DAL.Mongo;
+using Microsoft.Extensions.Logging;
+using NLog.Web;
+using System.IO;
+
+namespace Centaurus
+{
+    public class AlphaStartup : IStartup<AlphaSettings>
+    {
+        static Logger logger = LogManager.GetCurrentClassLogger();
+        private IHost host;
+
+        public void Run(AlphaSettings settings)
+        {
+            ConfigureConstellation(settings);
+
+            host = CreateHostBuilder(settings).Build();
+            _ = host.RunAsync();
+        }
+
+        public void Shutdown()
+        {
+            ConnectionManager.CloseAllConnections();
+            host.StopAsync().Wait();
+        }
+
+        #region Private Members
+
+        private void ConfigureConstellation(AlphaSettings settings)
+        {
+            Global.Init(settings, new MongoStorage());
+
+            Global.AppState.StateChanged += Current_StateChanged;
+
+            MessageHandlers<AlphaWebSocketConnection>.Init();
+        }
+
+        private void Current_StateChanged(object sender, ApplicationState e)
+        {
+            if (e == ApplicationState.Failed)
+            {
+                logger.Error("Application failed");
+
+                Shutdown();
+            }
+        }
+
+        private IHostBuilder CreateHostBuilder(AlphaSettings settings)
+        {
+            return Host.CreateDefaultBuilder()
+                .ConfigureLogging(logging =>
+                {
+
+                    logging.ClearProviders();
+                    logging.AddConsole();
+
+                    if (settings.Verbose)
+                        logging.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Trace);
+                    else if (settings.Silent)
+                        logging.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Error);
+                    else
+                        logging.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Information);
+                })
+                .UseNLog()
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.UseStartup<Startup>()
+                        .UseUrls(settings.AlphaUrl);
+                });
+        }
+
+        class Startup
+        {
+            public Startup(IConfiguration configuration)
+            {
+                Configuration = configuration;
+            }
+
+            public IConfiguration Configuration { get; }
+
+            // This method gets called by the runtime. Use this method to add services to the container.
+            // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+            public void ConfigureServices(IServiceCollection services)
+            {
+                services
+                    .AddMvc(options => options.EnableEndpointRouting = false);
```

Comment:
Do we need MVC here? Please check and if we don't use MVC packages, remove all references from the project.

Reply:
As mentioned above, we need it.